scripts/publish.ts

```diff
@@ -419,59 +427,55 @@ async function run() {
       path.resolve(rootDir, 'packages', pkg.packageDir, 'package.json'),
       async (config) => {
         await Promise.all(
-          (pkg.dependencies ?? []).map(async (dep) => {
+          Object.keys(config.dependencies ?? {}).map(async (dep) => {
             const depPackage = packages.find((d) => d.name === dep)
 
-            if (!depPackage) {
-              throw new Error(`Could not find package ${dep}`)
```

Comment:
throwing here is wrong as it fails the promise and `Promise.all` stops after the first rejection. We can just ignore dependencies that are _not_ dependencies we care about